src/command_modules/azure-cli-storage/azure/cli/command_modules/storage/tests/latest/test_storage_blob_scenarios.py

```diff
@@ -373,6 +373,22 @@ def test_storage_blob_append(self, resource_group, storage_account):
             self.storage_cmd('storage blob upload -c {} -f "{}" -n {} --type append --if-none-match *', account_info,
                              container, local_file, blob_name)
 
+    @ResourceGroupPreparer()
+    def test_storage_blob_update_service_properties(self, resource_group):
+        storage_account = self.create_random_name(prefix='account', length=24)
+
+        self.cmd('storage account create -n {} -g {} --kind StorageV2'.format(storage_account, resource_group))
+        account_info = self.get_account_info(resource_group, storage_account)
+
+        self.storage_cmd('storage blob service-properties show', account_info) \
+            .assert_with_checks(JMESPathCheck('staticWebsite.enabled', False))
+
+        self.storage_cmd('storage blob service-properties update --static-website --index-document index.html '
+                         '--404-document error.html', account_info) \
+            .assert_with_checks(JMESPathCheck('staticWebsite.enabled', True),
+                                JMESPathCheck('staticWebsite.errorDocument_404Path', 'error.html'),
+                                JMESPathCheck('staticWebsite.indexDocument', 'index.html'))
+
```

Comment:
Do we want to also do a test for updates to delete_retention_policy?  This test doesn't test enabling soft-delete and setting the days a soft-delete blob would be retained.